overview.md

```diff
@@ -2,12 +2,28 @@
 
 ## Important note
 
-The `AzureContainerAppsRC` task is the _Release Candidate (RC)_ version of the `AzureContainerApps` task that ships out
-of the box with Azure Pipelines, which can be found
+The `AzureContainerAppsRC` task found in this repository is the _Release Candidate (RC)_ version of the
+`AzureContainerApps` task that ships out of the box with Azure Pipelines, which can be found
 [here](https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks/AzureContainerAppsV0). The intended purpose
 of this Release Candidate task is to offer users immediate early access to features, bug fixes and patches that will
 eventually be rolled out in the official `AzureContainerApps` task at the end of every three week sprint.
 
+## Running this task on Microsoft-hosted agents
```

Comment:
Could you please clarify if at this point we still see some value in having both readme and overview files? Do you see some core differences between them?

Reply:
The contents that users see on the installation page come from the `overview.md` file, but you bring up a good point: I'm going to remove that file and have the installation page point to the content from the `README.md`. Good catch 👍